--- debian/backuppc/etc/backuppc/config.pl	2005-03-03 23:39:20.000000000 +0100
+++ debian/backuppc/etc/backuppc/config.pl	2005-03-03 18:55:02.000000000 +0100
@@ -41,7 +41,10 @@
 #
 # Host name on which the BackupPC server is running.
 #
-$Conf{ServerHost} = 'XXXXXX';
+$ENV{'PATH'} = '/bin:/usr/bin';
+delete @ENV{'IFS', 'CDPATH', 'ENV', 'BASH_ENV'};
+$Conf{ServerHost} = `hostname`;
+chomp($Conf{ServerHost});
 
 #
 # TCP port number on which the BackupPC server listens for and accepts
@@ -291,7 +297,7 @@
 # Full path to various commands for archiving
 #
 $Conf{SplitPath} = '/usr/bin/split';
-$Conf{ParPath}   = '';
+$Conf{ParPath}   = '/usr/bin/par2' if -x '/usr/bin/par2';
 $Conf{CatPath}   = '/bin/cat';
 $Conf{GzipPath}  = '/bin/gzip';
 $Conf{Bzip2Path} = '/bin/bzip2';
@@ -807,7 +810,7 @@
 #                to permanent media, such as tape, CDR or DVD.
 #               
 #
-$Conf{XferMethod} = 'smb';
+$Conf{XferMethod} = 'rsync';
 
 #
 # Level of verbosity in Xfer log files.  0 means be quiet, 1 will give
@@ -903,7 +906,7 @@
 #
 # This setting only matters if $Conf{XferMethod} = 'smb'.
 #
-$Conf{SmbClientPath} = '/usr/bin/smbclient';
+$Conf{SmbClientPath} = '';
 
 #
 # Command to run smbclient for a full dump.
@@ -1181,7 +1181,7 @@
 # version of rsync that is used on the server for rsync and rsyncd
 # transfers.
 #
-$Conf{RsyncBackupPCPath} = "";
+$Conf{RsyncBackupPCPath} = "/usr/bin/rsync_bpc";
 
 #
 # Ssh arguments for rsync to run ssh to connect to the client.
@@ -1269,7 +1269,6 @@
             '--hard-links',
             '--delete',
             '--delete-excluded',
-            '--one-file-system',
             '--partial',
             '--log-format=log: %o %i %B %8U,%8G %9l %f%L',
             '--stats',
@@ -1479,7 +1482,7 @@
 # Full path for ssh. Security caution: normal users should not
 # allowed to write to this file or directory.
 #
-$Conf{SshPath} = '/usr/bin/ssh';
+$Conf{SshPath} = '/usr/bin/ssh' if -x '/usr/bin/ssh';
 
 #
 # Full path for nmblookup. Security caution: normal users should not
@@ -1488,7 +1491,7 @@
 # nmblookup is from the Samba distribution. nmblookup is used to get the
 # netbios name, necessary for DHCP hosts.
 #
-$Conf{NmbLookupPath} = '/usr/bin/nmblookup';
+$Conf{NmbLookupPath} = '';
 
 #
 # NmbLookup command.  Given an IP address, does an nmblookup on that
@@ -1869,7 +1868,7 @@
 # rounded up (ie: 2.5 means a user will never receive email more
 # than once every 3 days).
 #
-$Conf{EMailNotifyMinDays} = 2.5;
+$Conf{EMailNotifyMinDays} = 1;
 
 #
 # Name to use as the "from" name for email.  Depending upon your mail
@@ -1931,8 +1934,8 @@
 #    $Conf{CgiAdminUsers}     = 'craig celia';
 #    --> administrative users are only craig and celia'.
 #
-$Conf{CgiAdminUserGroup} = '';
-$Conf{CgiAdminUsers}     = '';
+$Conf{CgiAdminUserGroup} = 'backuppc';
+$Conf{CgiAdminUsers}     = 'backuppc';
 
 #
 # TCP port number of the SCGI server.  A negative value disables the
@@ -1963,7 +1966,7 @@
 # Full URL of the BackupPC_Admin CGI script, or the configured path
 # for SCGI.  Used for links in email messages.
 #
-$Conf{CgiURL} = 'http://XXXXXX/cgi-bin/BackupPC_Admin';
+$Conf{CgiURL} = 'http://'.$Conf{ServerHost}.'/index.cgi';
 
 #
 # Full path to the rrdtool command.  If available, graphs of pool usage
<<<<<<< HEAD
=======
@@ -1181,7 +1181,7 @@
 # version of rsync that is used on the server for rsync and rsyncd
 # transfers.
 #
-$Conf{RsyncBackupPCPath} = "";
+$Conf{RsyncBackupPCPath} = "/usr/bin/rsync_bpc";
 
 #
 # Ssh arguments for rsync to run ssh to connect to the client.
@@ -1269,7 +1269,6 @@
             '--hard-links',
             '--delete',
             '--delete-excluded',
-            '--one-file-system',
             '--partial',
             '--log-format=log: %o %i %B %8U,%8G %9l %f%L',
             '--stats',
@@ -1869,7 +1868,7 @@
 # rounded up (ie: 2.5 means a user will never receive email more
 # than once every 3 days).
 #
-$Conf{EMailNotifyMinDays} = 2.5;
+$Conf{EMailNotifyMinDays} = 1;
 
 #
 # Name to use as the "from" name for email.  Depending upon your mail
>>>>>>> 315189c1e2f3ee71d9d3300951b6930900502c67
@@ -2060,7 +2059,7 @@
 # Security caution: normal users should not allowed to write to this file
 # or directory.
 #
-$Conf{RrdToolPath} = '';
+$Conf{RrdToolPath} = '/usr/bin/rrdtool';
 
 #   
 # Language to use.  See lib/BackupPC/Lang for the list of supported
